docs/how-to's/kuadrant-addon-walkthrough.md

```diff
@@ -0,0 +1,65 @@
+# Kuadrant operator addon
+
+## Introduction
+The following walkthrough will show you how to install/setup the Kuadrant operator via OCM (Open cluster management addons)
+
+## Prerequisites
+* Kind
+* OLM 
```

Comment:
What's the OLM prerequisite here?
A local dev dependency or something that local-setup does for you?

Reply:
its something the local setup does for you, looking at it again it doesn't make sense to add it if local setup will add it ill take it out